packages/modules/src/content/Content.ts

```diff
@@ -209,7 +209,7 @@ export function verifyDataStructure(input: IContent | PartialContent): void {
       }
     })
   }
-  DataUtils.validateId(input.schema, 'Identifier')
+  DataUtils.validateId(Identifier.getIdentifierKey(input.schema), 'Identifier')
```

Comment:
minor change. For now this is OK (as there are no users), but technically, this would be backward issue IMO.